_includes/tutorials/produce-consume-lang/scala/markup/dev/3_5-utils-implicits.adoc

```diff
@@ -0,0 +1,11 @@
+In consequence, we will also add a package object to hold all the necessary functions (CSV conversion, JSON Conversion ...)
+that are not directly related to our goal (produce and consume). Theses functions will increase the interactive aspect
+of the tutorial.
```

Comment:
```suggestion
Let's add some helper functions to the code making our application more interactive.
```

Reply:
I tried to insist on not focusing too much on this part. Maybe I'm the one focusing too much 😆!